spec/function.dd

```diff
@@ -1241,8 +1241,13 @@ int foo(in int x, out int y, ref int z, int q);
     $(TROW $(D scope), references in the parameter
     cannot be escaped (e.g. assigned to a global variable).
     Ignored for parameters with no references)
+    $(TROW $(D return), Parameter may be returned or copied to the first parameter,
```

Comment:
The LaTeX errors are caused by the last comma in this line. The comma is interpreted by ddoc as a parameter separator, which in LaTeX is translated into an column separator (ampersand) because you're in a table. To fix, you may want to use `$(TROW $(D return),  $(ARGS Parameter may be returned...))`